### PR TITLE
MAISTRA-2648: WASM fix for s390x

### PR DESCRIPTION
**Pull request by knm3000**
Fixes https://issues.redhat.com/browse/MAISTRA-2648
The permanent fix is not a part of maistra/envoy yet:
https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/198

### CODE REVIEW

**Review by jwendell — COMMENTED** [review]
LGTM

File: maistra/ci/common.sh [hunk 1]
```diff
@@ -60,3 +60,7 @@ sed -i "s|/work/|$(pwd)/|" maistra/vendor/proxy_wasm_cpp_sdk/toolchain/cc_toolch
 
 # modify include paths to avoid overly long gcc command line
 fix_include_paths
+
+# wasm fix for s390x https://issues.redhat.com/browse/MAISTRA-2648
+# the permanent fix is https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/198 (not a part of maistra/envoy yet)
+patch -p1 -i maistra/ci/proxy-wasm-cpp-host.patch
```
Comment (oschaaf) [inline maistra/ci/common.sh:66]: Bazel provides a way to patch in changes to dependencies as well; which might be a little cleaner?
We could set up `_proxy_wasm_cpp_host` to apply the patch over at
https://github.com/maistra/envoy/blob/0f592ba9b37a354a97da929722f0c70d7d85106e/bazel/repositories.bzl#L793 
WDYT?

**Review by oschaaf — COMMENTED** [review]
Just a thought about the way the required changes get patched in, the patch itself LGTM

File: maistra/ci/common.sh (hunk 1 above)
Comment (oschaaf) [reply maistra/ci/common.sh:66]: (following up on this would imply moving this PR over to maistra/envoy)

Comment (knm3000) [reply maistra/ci/common.sh:66]: Thanks for the review, I'll try to move the patch to maistra/envoy (and then update https://github.com/maistra/proxy/blob/maistra-2.1/WORKSPACE with the new envoy commit).

Comment (jwendell) [reply maistra/ci/common.sh:66]: It's not necessary to touch proxy after this. After your PR is merged on envoy side, automation takes place and will open a PR in proxy with the changes that got merged in envoy.

**Comment by knm3000** [comment]
Opened maistra/envoy PR https://github.com/maistra/envoy/pull/134 instead of this one.

**Pull request closed without merging**